### PR TITLE
Improvement of documentation: how to install pip if not already installed

### PR DESCRIPTION
**Pull request by ghost**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,16 +97,35 @@ https://github.com/pandas-dev/pandas
...
 ```sh
 # conda
 conda install -c conda-forge pandas
 ```
 
+If you want to use pip, you will first have to make sure that it is installed. Type:
+
+```sh
+# check that pip is installed
+pip --version
```
Comment (twoertwein) [inline README.md:111]: I think we should consistently use `python -m pip` in the documentation. It is used in many places but not in 
`doc/source/getting_started/index.rst` and `doc/source/getting_started/install.rst`

Comment (TimotheeMarreroMeyer) [reply README.md:111]: Hello I am new to github. Could you tell me the steps to do what you ask me? Do I have to commit again on the branch I created and make the change?

Comment (twoertwein) [reply README.md:111]: Yes, you have to commit to the branch you created: make changes, git add, git commit, git push

Comment (TimotheeMarreroMeyer) [reply README.md]: It’s normally good

**Comment by mroeschke** [comment]
Thanks for the PR, but I think the language here is okay as is. Having Python is an implicit prereq for installing pandas.

Closing but happy to have contributions towards good first issues https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by TimotheeMarreroMeyer** [comment]
Precisely it is not python that I check but pip because it is not always delivered with python.

**Pull request closed without merging**